Contest_API.md

```diff
@@ -1209,7 +1209,38 @@ Note that at least one of `from_team_id` and `to_team_id` has to be
 
 Clients with the `public` role can only view clarifications replies
 from the jury to all teams, that is, messages where both `from_team_id`
-and `to_team_id` are `null`.
+and `to_team_id` are `null`. Clients with the `team` role can only view
+their own clarifications (sent or received) and public clarifications.
+
+#### POST clarifications
+
+To add a clarification one can use the `POST` method on the clarifications endpoint.
+The `POST` must include a valid JSON object with the same attributes the clarification
+endpoint returns with a `GET` request with the following exceptions:
+
+* If the server supports a `team` role, `id`, `to_team_id`, `reply_to_id`, `time`, and
```

Comment:
I'd replace `If the server supports` with `When submitting using`, or at least add words along these lines. Now it is only implicit that this is when submitting as team role.

Reply:
Agreed, will do in following commit.